src/common/tools/read_task_planning.py

```
@@ -0,0 +1,48 @@
+        pattern = os.path.join("projects", project_name, "planning", "task-01*")
+        
+        # Find files matching the pattern
+        matching_files = glob.glob(pattern)
+        
+        # Check if any files were found
+        if not matching_files:
+            return f"Error: No files matching pattern '{pattern}' found"
+        
+        # If multiple files match, use the first one and warn
+        if len(matching_files) > 1:
+            print(f"Warning: Multiple files match pattern '{pattern}'. Using the first one: {matching_files[0]}")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)